server_admin/topics/admin-console-permissions/fine-grain.adoc

```diff
@@ -43,20 +43,20 @@ The first thing we must do is login to the Admin Console so we can set up permis
 of the client, we want to define fine-grain permissions for.
 
 .Client management
-image:{project_images}/fine-grain-client.png[]
+image:{project_images}/fine-grain-client.png[Fine grain client]
```

Comment:
Text on this screen is hard to read.  Can you shrink this screen by a quarter and recapture it?